tests/test_config_parsing/test_items_equal.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pytest
+from hypothesis import given
+
+from ert._c_wrappers.enkf import ConfigKeys, ResConfig
+
+from .config_dict_generator import config_dicts, to_config_file
+
+
+@pytest.mark.usefixtures("use_tmpdir", "set_site_config")
+@given(config_dicts())
+def test_items_from_config_dict_same_as_from_file(config_dict):
+    cwd = os.getcwd()
+    filename = config_dict[pytest.TEST_CONFIG_FILE_KEY]
+    to_config_file(filename, config_dict)
+    config_dict[ConfigKeys.CONFIG_DIRECTORY] = cwd
+    assert (
+        ResConfig(config_dict=config_dict).license_root_path
+        == ResConfig(filename).license_root_path
+    )
+    assert ResConfig(config_dict=config_dict).job_list == ResConfig(filename).job_list
+    assert ResConfig(config_dict=config_dict).env_vars == ResConfig(filename).env_vars
```

Comment:
this is a comment really related to `config_dict_generator.py` - I see that we generate stuff for install job (`INSTALL_JOB`), install job dir (`INSTALL_JOB_DIRECTORY`), and we create the files given under install job, and we create the directories given under install job directory, but we don't create any files in the directories. perhaps you could add that, for more realistic config dicts?

also i see that we do not add `SET_ENV` in the config dict generator! could you please add that one, too?

Reply:
We do add some SETENV stuff

Reply:
yes we do, indeed - what about `SET_ENV`?!?!? ; )

Reply:
but i didn't miss anything on contents of the install job dirs, did i? that point was valid?

Reply:
Is SET_ENV a thing as well?

Reply:
I'm working on the other thing.

Reply:
I've done gone did some smarts me thinks, take a looksie

Reply:
no sorry again my silly attempt at a joke - I thought it was `SET_ENV`, looked for it, didn't find nuthin, and commented. but it turns out there is no `SET_ENV` O : 

i likes the smarts, thanks a lots!